### ISSUE TITLE
Option to select primary CP builder ability

### ISSUE THREAD

**Issue by Warkonaut** [wowsims/wotlk#2742]
**Problem**
Cannot sim Combat Dagger rogue, using Backstab instead of Sinister Strike. The sim keeps using Sinister Strike, even if dagger is in MH, have glyph and talents for Backstab.

**Solution**
Please add a setting for primary CP builder (Sinister Strike/Backstab/Hemo).

**Alternate solution**
CP builder ability is chosen automatically based on selected weapons and talents. This would be harder to do right. Dropdown menu for manual selection is probably better.

**Additional context**
Combat Daggers is worse than Combat Fist & Dagger, but I got i226 first weapon and i252 slow dagger, in this case the dagger & Backstab will be most likely much better; but cannot sim it right now.

**Comment by where-fore** [comment]
If it helps in the meantime, Simon's sheet loosely supports backstab building, and with a P1 default 60s fight, swapping Calamity's/Sinister Strike for Fang of Oblivion/Backstab is a 100 dps loss, going up to 170 dps at 180s. You generally don't have the combo points to maintain Rupture at 5cp with Backstab.

**Comment by Warkonaut** [comment]
You are absolutely right about the CP generation. However, there are glyphs like Glyph of Rupture/Backstab that prolong the Rupture dot (up to 10 secs in total) + possibilities to improve the energy and CP generation by moving few talent points out of combat tree.

Builds like this seem to generate enough CP: https://www.wowhead.com/wotlk/talent-calc/rogue/00532300503-00520510500350102231005012-502_001qf011qf121qf8

What about PvE Subtlety builds for Algalon? They use Hemo or Backstab...
